src/main/java/com/minecolonies/coremod/colony/ColonyView.java

```diff
@@ -64,6 +64,14 @@
      */
     private Set<Block> freeBlocks = new HashSet<>();
 
+    /**
+     * The Set of waypoints.
+     */
+    private Set<BlockPos> wayPoints = new HashSet<>();
+
+    /**
+     * The overall happiness of the colony.
+     */
     private double overallHappiness = 5;
```

Comment:
We have that around at some places but its not on the client and I think it would be bad style if we'd call not client classes from client classes.
But then I'd have to recreate the same thing again here.